.config/ansible-lint.yaml

```
@@ -3,3 +3,5 @@
 profile: production
 skip_list:
   - meta-runtime[unsupported-version]  # 2.14 deemed too old by ansible-lint
+exclude_paths:
+  - .yamllint.yaml
```

```suggestion
  - .ansible  # created by ansible-compat
```

(Took me a while to hunt down, but this is due to [ansible-compat#439](https://github.com/ansible/ansible-compat/pull/439/files#diff-1aa5764fb9e6d74d6a723ce1662397e6800caae433197d2c982e1299550c687c): newer versions install dependencies in the project directory, instead of ~/.cache/ansible-compat.)

---

As we talked about at the meeting, I added the cache from ansible-compt to gitignore via https://github.com/Tina-pm/ansible-playbooks/compare/c353ca926555eb9524d91b28f5e488eed4ddc5c9..29b8e3e1302d975a62acd20453244aa25c5bf196